### PR TITLE
MultiThreadExecutor number of threads is at least 2+ in default.

### PR DESCRIPTION
**Pull request by fujitatomoya**
minor comment follow-up for https://github.com/ros2/rclcpp/pull/2030

Signed-off-by: Tomoya Fujita <Tomoya.Fujita@sony.com>

### CODE REVIEW

**Comment by fujitatomoya** [comment]
@clalancette minor doc fix, can you check English? 😄

File: rclcpp/include/rclcpp/executors/multi_threaded_executor.hpp [hunk 1]
```diff
@@ -47,7 +47,7 @@ class MultiThreadedExecutor : public rclcpp::Executor
    *
    * \param options common options for all executors
    * \param number_of_threads number of threads to have in the thread pool,
-   *   the default 0 will use the number of cpu cores found instead
+   *   the default 0 will use the number of cpu cores found at least 2+
```
Comment (clalancette) [inline rclcpp/include/rclcpp/executors/multi_threaded_executor.hpp:50]:
```suggestion
   *   the default 0 will use the number of cpu cores found (minimum of 2)
```

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rclcpp/include/rclcpp/executors/multi_threaded_executor.hpp (hunk 1 above)
Comment (fujitatomoya) [reply rclcpp/include/rclcpp/executors/multi_threaded_executor.hpp:50]: @clalancette 👍 thanks for checking, i will change it as suggested.

**Review by fujitatomoya — COMMENTED** [review]
address https://github.com/ros2/rclcpp/pull/2032#discussion_r1006824604

**Review by clalancette — APPROVED** [review]

**Review by alsora — APPROVED** [review]

**Comment by fujitatomoya** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/17555/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/12093/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/18122/)

**Pull request merged**